features/step_definitions/needs_steps.rb

```diff
@@ -0,0 +1,15 @@
+When(/^I start editing the needs from the .*? page$/) do
+  click_on 'Add/remove needs'
+end
+
+When(/^I choose the first need in the dropdown$/) do
+  option = first('#need_ids option').text
+  select option, from: 'need_ids'
+  click_button 'Save needs'
+end
+
+Then(/^I should see the first need in the list of associated needs$/) do
+  page.find("h2:contains('Attachments')")
```

Comment:
It was checking whether we were on the right page (since that would have "Attachments"). But that could change so I'm removing it.